app/models/claim.rb

```diff
@@ -115,6 +117,17 @@ def submit!
     end
   end
 
+  def approve!(approved_by:)
+    if approvable?
+      update(
+        approved_at: DateTime.now,
```

Comment:
Can we make this consistent with the `submit!` method above and use `Time.zone.now`? There is a subtle difference between the two values: one accounts for the configured timezone in the app, the other uses the server's configured timezone, which may not be the same.